bsip-0075.md

```diff
@@ -15,28 +15,85 @@ Worker: TBD
 
-This new change shall permit the asset owner to determine *at asset creation time* who should have the responsibility to maintain those values.
+This new change shall permit the asset owner to determine who should have the responsibility to maintain those values.  It shall also permit the asset owner to forfeit the right to update the parameter.  Therefore the asset owner may continue changing the responsibility unless and until the permission is forfeited.
+
 
 # Specification
-The `asset_object.asset_options.issuer_permissions` shall contain the new `issuer_fed_mcr` permission that determines if an issuer can change the related `flags` value.
 
-The `asset_object.asset_options.issuer_permissions` shall contain the new `issuer_fed_mssr` permission that determines if an issuer can change the related `flags` value.
+**Note:** "Implementation Hints" are not to be considered part of the formal specification, but merely as a _possible_ implementation.
+
+## Parameter
+
+Each of these new issuer-defined parameters shall be independently configurable of the other.
+
+**Implementation Hint:** New optional fields in `bitasset_options` may be defined for every asset to contain these new parameters
+
+```
+struct bitasset_options
+{
+	struct ext {
+      //...
+      /// The parameters which the issuer has permission to update
+      optional<uint16_t> mcr; // Maintenance Collateral Ratio
+      optional<uint16_t> mssr; // Maximum Short Squeeze Ratio
+    }
+};
+```
+
+## Effect of a Parameter
+
+All relevant blockchain contracts _shall use_ a parameter _if it is set_ and _shall ignore_ the derived parameters from the asset's published feeds.
+
+All relevant blockchain contracts _shall ignore_ a parameter _if it is not set_ and _shall use_ the derived parameters from the asset's published feeds.
+
+At the time that this BSIP activates, these issuer-defined parameters shall not be _set_ and will therefore not affect the current behavior of the asset _unless and until_ each asset owner sets a parameter.
+
+An asset owner shall also be to set and unset a parameter at will unless the permission to change that parameter has been forfeited as described in [Permission to Change the Parameter](#permissions).
+
+
+## <div id="permissions">Permission to Change the Parameters</div>
+
+The permission to change any of these parameters shall make use of new permissions.  _Each of the parameters shall have its own distinct permission._  Disabling the permission prohibits the changing of the parameter as described in [Changing the Parameters](#changing-parameters).
 
-The `create_asset_operation` shall be modified to include the setting of the parameters above, as well as initial `flags` values and ratios for MCR and MSSR.
+The existing set of asset issuer permissions shall be expanded to permit an asset owner to forfeit changing the parameters.  It shall not be permitted to regain a permission after it is forfeited.
```

Comment:
Regaining is possible when supply is zero.

Reply:
I see, that is reasonable